algorithmic_efficiency/workloads/imagenet_resnet/imagenet_v2.py

```diff
@@ -37,7 +37,7 @@ def _decode_example(example: Dict[str, float]) -> Dict[str, float]:
                                                resize_size)
     return {'inputs': image, 'targets': example['label']}
 
-  ds = ds.map(_decode_example, num_parallel_calls=16)
```

Comment:
can we undo this change? or is it needed for some reason?

Reply:
Sorry, I encountered a weird problem with `num_workers` for PyTorch and this was one of my attempts at debugging. I will undo this line.

Reply:
Fixed!